src/main/java/de/tum/cit/aet/artemis/exam/domain/room/ExamRoom.java

```diff
@@ -0,0 +1,152 @@
+package de.tum.cit.aet.artemis.exam.domain.room;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+import org.springframework.context.annotation.Conditional;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+
+import de.tum.cit.aet.artemis.core.domain.AbstractAuditingEntity;
+import de.tum.cit.aet.artemis.exam.config.ExamEnabled;
+import de.tum.cit.aet.artemis.exam.dto.room.ExamSeatDTO;
+
+@Conditional(ExamEnabled.class)
+@Entity
+@Table(name = "exam_room")
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class ExamRoom extends AbstractAuditingEntity {
+
+    /**
+     * The room number, e.g. 123.EG.01, or 123.456.78.9
+     */
+    @Column(name = "room_number", nullable = false, length = 50)
+    private String roomNumber;
+
+    /**
+     * An alternative room number, if it exists. Used to improve auto-complete.
+     */
+    @Column(name = "alternative_room_number", nullable = true, length = 50)
+    private String alternativeRoomNumber;
+
+    /**
+     * The name of the exam room.
+     */
+    @Column(name = "name", nullable = false, length = 255)
+    private String name;
+
+    /**
+     * An alternative name for the exam room, doesn't need to exist. Used to improve auto-complete.
+     */
+    @Column(name = "alternative_name", nullable = true, length = 255)
+    private String alternativeName;
```

Comment:
Can you add examples to this values in the javadoc. I assume this would be

roomNumber = 5602.EG.001
alternativeRoomNumber = 00.02.001
name = Friedrich L. Bauer Hörsaal
alternativeName = MI HS1

correct?

Does it make sense for all rooms to only have one number / name and one alternative? Like in this case just "HS1" or something as an additional name alternative?